tests/dummy/app/templates/freestyle.hbs

```diff
@@ -31,4 +31,21 @@
 
   {{/freestyle-section}}
 
+  {{#freestyle-section name="CTA"}}
```

Comment:
TODO: Use `freestyle-collection` + `freestyle-variant` to display the different variants